README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
+* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
```

Comment:
I suppose there needs to be a better way to explain the purpose of why the question is unique and the goal of the question. There are many people who won't know WAI-ARIA, nor WCAG. The first question is generic: what does one know about accessibility? The second question is specific to (1) do you have experience with WAI-ARIA and (2) based on getting feedback on what they might know about accessibility in general, can they explain the differences between accessibility (foundational HTML techniques via WCAG, Section 508, etc.) and WAI-ARIA which is very specific to supporting assistive technologies such as screen reader usage (that is not so much about accessibility, as much as accommodation, user experience and use cases for user interface design patterns that need to be linearized or modalized for use with a specific product).